Packs/GigamonThreatINSIGHT/Integrations/GigamonThreatINSIGHT/README.md

```diff
@@ -11,312 +11,7781 @@ For more information about the Cortex XSOAR integration visit the Insight help d
+                "subdivision": "CA",
+                "tags": [
+                    "Training Modern",
+                    "ESXI 7.0"
+                ],
+                "updated": "2022-05-16T15:11:03.400Z"
+            }
+        ]
+    }
+}
+```
 
-None
+#### Human Readable Output
 
-### Insight-get-saved-searches
+>### Results
+>|account_code|admin|city|country|created|disabled|interfaces|location|pcap_enabled|sensor_id|serial_number|status|subdivision|tags|updated|
+>|---|---|---|---|---|---|---|---|---|---|---|---|---|---|---|
+>| gdm |  |  |  | 2021-12-17T20:40:54.348Z | 2022-03-28T18:18:46.826Z |  |  | false | gdm1 |  |  |  |  | 2021-12-17T20:40:54.348Z |
+>| gdm |  |  |  | 2022-03-28T18:17:37.696Z |  |  |  | false | gdm2 |  |  |  |  | 2022-03-28T18:17:37.696Z |
+>| tma |  | Santa Clara | US | 2022-05-02T19:45:36.987Z |  |  | latitude: 37.3541069<br/>longitude: -121.955238 | false | tma2 |  |  | CA | Training Modern,<br/>ESXI 7.0 | 2022-05-16T15:11:03.400Z |
 
-Get user's saved searches.
 
-Example Command
-`!insight-get-saved-searches`
+### insight-get-devices
+***
+Get a list of all devices.
 
-#### Arguments
 
-None
+#### Base Command
 
-### Insight-get-sensors
+`insight-get-devices`
+#### Input
 
-Get a list of all sensors.
+| **Argument Name** | **Description** | **Required** |
+| --- | --- | --- |
+| start_date | Filter devices based on when they were seen. | Optional | 
+| end_date | Filter devices based on when they were seen. | Optional | 
+| cidr | Filter devices that are under a specific CIDR. | Optional | 
+| sensor_id | Filter devices that were observed by a specific sensor. | Optional | 
+| traffic_direction | Filter devices that have been noted to only have a certain directionality of traffic ("external" vs "internal"). | Optional | 
+| sort_by | Sort output by: "ip", "internal", "external". | Optional | 
+| sort_direction | Sort direction ("asc" vs "desc"). | Optional | 
 
-Example Command
-`!insight-get-sensors account_uuid=0a7dae9g-6f74-4c75-78ef-856483763e1d4`
 
-#### Arguments
+#### Context Output
 
-| Name | Description | Example |
-| ---- | ----------- | ------- |
-| account_uuid | UUID of account to filter by. | `0a7dae9g-6f74-4c75-78ef-856483763e1d4` |
-| account_code | Account code to fiilter by. | `abc` |
-| sensor_id | ID of the sensor to filter by. | `abc1` |
+| **Path** | **Type** | **Description** |
+| --- | --- | --- |
+| Insight.Devices.date | date | Date when the device was first seen | 
+| Insight.Devices.external | boolean | If external traffic has been observed for this device | 
+| Insight.Devices.internal | boolean | If internal traffic has been observed for this device | 
+| Insight.Devices.ip_address | string | IP address of the device | 
+| Insight.Devices.sensor_id | string | ID code of the sensor | 
 
-### Insight-get-devices
+#### Command example
+```!insight-get-devices start_date=2019-01-01T00:00:00.000Z end_date=2019-01-31T23:59:59.999Z```
+#### Context Example
```

Comment:
Not sure how these Context Examples get in the README.md file (automatic vs manual) but the examples are far too large and need to be reduced for each command
e.g. for Insight-get-devices, two or at most three devices needs to be shown to the user for them to get the idea of the data returned. The example data shown is for about 100 devices.

Reply:
The commands in the command_example file get called and the result is paced in the readme file. It would be good to have a sample command for each command that does not generate a big result.